src/pybind/mgr/volumes/module.py

```diff
@@ -223,30 +223,43 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'perm': 'r'
         },
         {
-            'cmd': 'nfs export create '
-            'name=type,type=CephString '
+            'cmd': 'nfs export create cephfs '
             'name=fsname,type=CephString '
-            'name=binding,type=CephString '
             'name=attach,type=CephString '
+            'name=binding,type=CephString '
             'name=readonly,type=CephBool,req=false '
             'name=path,type=CephString,req=false ',
             'desc': "Create a cephfs export",
             'perm': 'rw'
         },
         {
-            'cmd': 'fs nfs export delete '
-                   'name=export_id,type=CephInt,req=true ',
+            'cmd': 'nfs export delete '
+                   'name=attach,type=CephString '
+                   'name=binding,type=CephString ',
             'desc': "Delete a cephfs export",
             'perm': 'rw'
         },
         {
             'cmd': 'nfs cluster create '
                    'name=type,type=CephString '
                    'name=clusterid,type=CephString '
-                   'name=size,type=CephInt,req=false ',
+                   'name=placement,type=CephString,req=false ',
             'desc': "Create an NFS Cluster",
             'perm': 'rw'
         },
+        {
+            'cmd': 'nfs cluster update '
+                   'name=clusterid,type=CephString '
+                   'name=placement,type=CephString ',
+            'desc': "Updates an NFS Cluster",
+            'perm': 'rw'
+        },
+        {
+            'cmd': 'nfs cluster delete '
+                   'name=clusterid,type=CephString ',
+            'desc': "Deletes an NFS Cluster",
+            'perm': 'rw'
+        },
```

Comment:
In addition to `ceph nfs cluster list`, we also should find a way to support;

`ceph nfs cluster info foo`: get the nfs servers' ip/ports. Otherwise, how does the user know where they were scheduled and what ip/port they are listening on?

cc @mgfritch @sebastian-philipp  is this information about the ip/port available somehow through the orchestrator?